src/Components/Shared/src/WebEventData.cs

```diff
@@ -79,11 +79,12 @@ private static EventArgs ParseEventArgsJson(ulong eventHandlerId, string eventAr
             {
                 throw new InvalidOperationException($"There was an error parsing the event arguments. EventId: '{eventHandlerId}'.", e);
             }
+
         }
 
-        private static T Deserialize<T>(string json) => JsonSerializer.Deserialize<T>(json, JsonSerializerOptionsProvider.Options);
+        static T Deserialize<T>(string json) => JsonSerializer.Deserialize<T>(json, JsonSerializerOptionsProvider.Options)!;
```

Comment:
```suggestion
        private static T Deserialize<T>(string json) => JsonSerializer.Deserialize<T>(json, JsonSerializerOptionsProvider.Options)!;
```

Reply:
Why do we need `!` here?

Reply:
All of the results returned by S.T.J are nullable. The JSON `null` value can always be substituted for any JSON serialized value. We just don't expect it in our case.